src/sentry/plugins/sentry_mail/models.py

```diff
@@ -235,6 +235,10 @@ def notify(self, notification):
 
         context = {
             'project_label': project.get_full_name(),
+            'project_link': absolute_uri('/{}/{}/'.format(
```

Comment:
so first of all, it looks like this class has a `get_project_url` method that you could use. 

second of all, it looks like this bug was introduced with [this change](https://github.com/getsentry/sentry/commit/d10e0d5187622aab6b6051882a52ec065dd81b42#diff-8c55693c5c3b7b812f03dc8a2a6fbac8L21). Since I your change would affect the context for all emails, is there a reason you're not just updating [this template](https://github.com/getsentry/sentry/blob/master/src/sentry/templates/sentry/emails/error.html#L131)?

Reply:
Ah, I was just trying to be consistent with what seemed to used to be done. For instance user report in this same file has a `project_link` that builds it exactly like this (I just copied and paste). I figured `project_link` was a convention, because that's what the user report emails did and what the alert emails used to do. So, I thought it would just be easiest to pass that information along and remain consistent with the rest of the codebase.

Reply:
https://github.com/getsentry/sentry/blob/master/src/sentry/plugins/sentry_mail/models.py#L363

Reply:
is that context even used anymore? it doesn't look like it is in this template: https://github.com/getsentry/sentry/blob/master/src/sentry/templates/sentry/emails/activity/new-user-feedback.html

also, from searching the codebase for `project_link`, it doesn't really look like that's used anymore. I can only find the instance you're trying to fix and [this](https://github.com/getsentry/sentry/blob/master/src/sentry/templates/sentry/emails/group_header.html#L4), where it looks like it's actually defined in the template, so the context isn't relevant. So, to me it seems like that was something we used to do and have since moved away from.